src/plugins/telemetryTable/components/TableConfiguration.vue

```diff
@@ -112,6 +112,15 @@ export default {
   },
   methods: {
     updateHeaders(headers) {
+      // add name column if it doesn't exist,
+      // it's always the first column when it's manually added
+      if (!headers.name) {
```

Comment:
A structured clone assigned to the var is better than the spread operator due immutability/performance

Reply:
Hey @evenstensberg ! thanks for the feedback, in this case I'm using the spread operator mainly to put name first in the headers, which is important. It also maintains immutability (just reassigning locally). The performance hit for such a small object that isn't nested is negligible and it's doing the main thing I need.